test/extended/util/annotate/rules.go

```diff
@@ -34,6 +34,11 @@ var (
 			`should answer endpoint and wildcard queries for the cluster`, // currently not supported by dns operator https://github.com/openshift/cluster-dns-operator/issues/43
 			// https://bugzilla.redhat.com/show_bug.cgi?id=1902702 - [sig-auth][Feature:LDAP][Serial] ldap group sync can sync groups from ldap: oc cp over non-existing directory/file fails
 			`\[sig-auth\]\[Feature:LDAP\]\[Serial\] ldap group sync can sync groups from ldap`,
+
+			// https://bugzilla.redhat.com/show_bug.cgi?id=1906518
+			// It is already disabled in o/k but because openshift/csi suite prefixes the test name
+			// it no longer matches there.
+			`.*\[Feature:VolumeSnapshotDataSource\].*`,
```

Comment:
the tests are still being run https://prow.ci.openshift.org/view/gs/origin-ci-test/pr-logs/pull/25764/pull-ci-openshift-origin-master-e2e-gcp-csi/1339208077420793856